content/en/blog/posts/measuring-progress:-a-product-maturity-model-for-digital-government.md

```diff
@@ -1,6 +1,4 @@
 ---
-aliases:
-  - "/2024/11/04/measuring-progress-a-product-maturity-model-for-digital-government//"
```

Comment:
These 2 lines need to be kept in the file in order for the redirect to work and maybe add a new line for the November 5th link
```
/2024/11/05/measuring-progress-a-product-maturity-model-for-digital-government/.
```